.eslintrc.json

```diff
@@ -63,5 +63,17 @@
     "@typescript-eslint/no-unsafe-return": "off",
     "@typescript-eslint/no-unsafe-argument": "off",
     "@typescript-eslint/no-unsafe-call": "off"
-  }
+  },
+  "overrides": [
+    {
+      "parser": "espree",
+      "files": [
+        "*.mjs"
+      ],
+      "parserOptions": {
+        "sourceType": "module",
+        "ecmaVersion": "latest"
```

Comment:
This was to get eslint to be happy about our rollup config (not strictly necessary, but ew! redlines!) even latest isn't latest enough yet for the json import assertion, but at least that's the only error left.